_posts/2018-08-22-next-stop-rome.md

```diff
@@ -0,0 +1,35 @@
+
+**What is Rustfest again?** It's a two day conference involving one day of workshops, and one day of talks! We aim for a diverse set of talks: technical and non-technical, advanced and beginner friendly content, live coding, panel, or slide based. If you like Rust, the community, or just hacking in general we aim to have something to appeal to you. This year the conference will host up to 350 awesome folks. It occurs bi-annually in a different city each edition.
+
+**Want to attend?** Pencil the date in your calendar! Like previous editions, we'll release a few ticket batches to ensure you get a chance to get your ticket. In the meantime, why not explore hotel/AirBNB options as well as things to do? Check out the [venue page](https://rome.rustfest.eu/location/) for more.
+
+**Need to convince your boss?** Some talking points: RustFest is the leading non-US based conference focused on Rust and the community around it. We are a community conference with (very) affordable tickets, and we can issue invoices to companies. 
+
+**What is it like?** Talks can be found online for [Berlin](https://www.youtube.com/watch?v=fI4RG_uq-WU&list=PL85XCvVPmGQh8nWR_Z-fTmPGsUWuzb-dn), [Київ](https://www.youtube.com/watch?v=AHprJNUCgQ0&list=PL85XCvVPmGQhvs1Rnet_24B-AI3YSM2YG), [Zürich](https://www.youtube.com/watch?v=jywiVWKm1TI&list=PL85XCvVPmGQj9mqbJizw-zi-EhcpS5jTP) and [Paris](https://www.youtube.com/watch?v=23lRkdDXqY0&list=PL85XCvVPmGQgdqz9kz6qH3SI_hp7Zb4s1). The conference is much more than that though, and you'll have the opportunity to meet people who work on some of the projects and efforts you know and love, share in fine cheese and bread, and drink world famous wine with your new and old friends.
+
+**Want to speak?** The [Call for Proposals](https://cfp.rustfest.eu/) is open! We would love to hear your proposal for a talk, workshop or discussion. Make sure to submit before **Date** As always, we'll be prioritizing talks not yet recorded at Rust conferences.
+
+> Note this is a **hard deadline** this year, we will not be able to extend as timelines are already very tight. **Submit early**.
+
+**Haven't spoken before but want to?** Submit! It's totally okay! If we accept your talk in our (blind) selection procedure we're happy to be your first experience. If you'd like, we're also happy to mentor you to give you some tips.
+
+**Want to volunteer?** It's great to have a few locals who'd like to help attendees enjoy the experience. If you'd like to help guide a cohort of fellow hackers to your favorite restaurant, give directions, or just generally help out and make a bunch of awesome friends please [let us know here](https://github.com/RustFestEU/blog.rustfest.eu/issues/12).
+
+**Barriers to participating?** Not all of us can easily attend these events for a variety of reasons. We'll be working with [Diversity Tickets](https://diversitytickets.org/) again. If the ticket cost itself is your only barrier, let us know.
```

Comment:
Maybe link to the new "infos & policies" page as info for other barriers we try to handle?